src/main/java/socialite/ui/PersonCard.java

```diff
@@ -155,6 +155,32 @@ private void makeRemark(Remark remark) {
         }
     }
 
+    // credits for this method goes to https://stackoverflow.com/questions/32781362/centering-an-image-in-an-imageview
+    private void centerImage(ImageView imageView) {
+        Image img = imageView.getImage();
+        if (img != null) {
```

Comment:
Great solution! One minor way that you can consider improving it from a Code Quality POV: Use guard clause for the "unusual case" instead. 

E.g.: Deal with `if (img == null) { return; }` which should make the "happy path" prominent as a large chunk of the code [lines 162-179] will no longer be indented by one level.

Reply:
Sorry for the late reply, I missed this out! Fixed in the latest commit!